### ISSUE TITLE
Add notification feature when scoring a goal

### ISSUE THREAD

**Issue by derycktse** [ManrajGrover/football-cli#15]
Thought that it will be nice when a goal happen, the cli can push a terminal notification. Is it possible?

**Comment by ManrajGrover** [comment]
@derycktse Yes, this is something that is possible. There are packages for it, for example [node-notifier](https://github.com/mikaelbr/node-notifier).

But where should we get the information from is the question. The event api is currently not [documented](http://api.football-data.org/docs/v1/index.html#_event_api). Maybe we would have to look into other free api alternatives or scrape it in the background.

**Comment by derycktse** [comment]
There is an awful way with current api:
Maybe we can store the score. Next time when we got the new result,  compare with previous one. If the score is change , there must be a goal.

**Comment by MCodez** [comment]
There is a Twitter Handle @GoalFlash. According to goal.com initiative: @GoalFlash will let you know when a team has scored within three seconds of the ball hitting the back of the net. Twitter API will do the rest of work. But, we need to take care of time of tweets and compare them so that we can confirm that the tweet read by API is latest. The tweet will contain all the information of the goal. We can try this.
